telemetry/specs/INT.mdk

```diff
@@ -384,12 +384,39 @@ ignore the designated bit position.)
 * TCP/UDP destination Port field: A port number in layer 4 will be reserved to
 indicate the existence of INT after TCP/UDP. INT source changes the destination
 Port, and sink must restore the original port number.
+* New Probe Marker fields: arbitrary 64-bit values are inserted after IP TCP/UDP 
+header to indicate the existence of INT after TCP/UDP. These fields should be 
+interpreted as unsigned integer values, stored in network byte order and are 
+initialized to a configured value.  This Probe marker is a variation of an early 
+IETF draft with existing implementations. More information can be found at 
+https://tools.ietf.org/html/draft-lapukhov-dataplane-probe-01.
```

Comment:
Please change the last two sentences as follows:

> This Probe marker is a variation of an early IETF draft with existing implementations [^DPP].
> [^DPP] :  Data-plane probe for in-band telemetry collection,  https://tools.ietf.org/html/draft-lapukhov-dataplane-probe-01

Reply:
I am adding the 2 [^DPP], assume it's the pdf format.